### DISCUSSION TITLE
How do I prevent the Colab from autoupdating the repository?

### DISCUSSION THREAD

**Discussion by JeleLele3** [TheLastBen/fast-stable-diffusion#1844] [Q&A] [answered]
I'm practically inept at programming. It's almost like Chinese to me.

If I delete !git pull from the startup code, the whole thing breaks and it fails to start. Any proper way to do it?

**Comment by TheLastBen** [comment, marked as answer]
the repo needs to stay up to date always, (unfortunatly)

**Reply by JeleLele3** [reply]
Thanks for the response!
